setup.py

```diff
@@ -23,19 +23,19 @@
     author_email="aca-it@uw.edu",
     include_package_data=True,
     install_requires=[
-        'Django==1.6',
-        'South',
+        'Django==1.10.2',
         'ordereddict',
         'simplejson',
-        'django-compressor<2.0',
+        'django-compressor',
         'BeautifulSoup',
         'urllib3==1.10.2',
         'django-templatetag-handlebars',
-        'Django-UserService==1.0.2',
+        'django-userservice==1.2.1',
         'unittest2',
         'AuthZ-Group',
         'python-binary-memcached',
         'UW-RestClients',
+        'PermissionsLogging',
```

Comment:
This isn't an app requirement, right?  This is a project level requirement.  On our hosts that's managed through a separate ansible component.